website/blog/2023-07-12-release-1.2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Release Notes - Podman Desktop 1.2
+description: Podman Desktop 1.2 has been released!
+slug: podman-desktop-release-1.2
+authors: [cdrage]
+tags: [podman-desktop, release, kubernetes, openshift]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.2 Release! 🎉
+
+We're excited to announce the release of Podman Desktop version 1.2.0! This release includes several new features, bug fixes, and improvements to enhance your container management experience. Here's a summary of the key changes in this release:
+
+- **Protocol handler support**: Added support for protocol handlers such as `open podman-desktop:extension/myextension`
```

Comment:
Can we focus on the 'why', or what it does for users? I think many users won't understand what this means.

Also, can you move it to the last position? Usually the top few things are the most significant for users, which IMHO is compose or kube context, then renaming.